modules/node_modules/@ncigdc/components/ClinicalCard.js

```diff
@@ -92,99 +92,121 @@ const ClinicalCard = ({
+                      {
+                        th: 'Days to Last Known Disease Status',
+                        td: x.days_to_last_known_disease_status,
+                      },
+                      { th: 'Days to Recurrence', td: x.days_to_recurrence },
+                      { th: 'Last Known Disease Status', td: x.last_known_disease_status },
+                      { th: 'Morphology', td: x.morphology },
+                      { th: 'Primary Diagnosis', td: x.primary_diagnosis },
+                      { th: 'Prior Malignancy', td: x.prior_malignancy },
+                      { th: 'Progression or Recurrence', td: x.progression_or_recurrence },
+                      { th: 'Site of Resection of Biopsy', td: x.site_of_resection_or_biopsy },
+                      { th: 'Tissue or Organ of Origin', td: x.tissue_or_organ_of_origin },
+                      { th: 'Tumor Grade', td: x.tumor_grade },
+                      { th: 'Tumor Stage', td: x.tumor_stage },
+                      { th: 'Vital Status', td: x.vital_status },
+                    ]}
+                    style={{ flex: 1 }}
+                  />
+                  <div
+                    style={{
+                      padding: '1rem',
+                      color: theme.greyScale7,
+                      fontSize: '2rem',
+                      lineHeight: '1.4em',
+                    }}
+                  >
+                    Treatments
+                  </div>
+                  {x.treatments && !!x.treatments.length &&
+                    <Table
+                      headings={[
+                        <Th key="id">ID</Th>,
+                        <Th key="agents">Therapeutic Agents</Th>,
+                        <Th key="intent_type">Treatment Intent Type</Th>,
+                        <Th key="therapy">Treatment or Therapy</Th>,
+                        <Th key="days">Days to Treatment</Th>,
+                      ]}
+                      body={
+                        <tbody>
+                          {x.treatments.map(t =>
+                            <Tr key={t.treatment_id}>
+                              <Td>{t.treatment_id || '--'}</Td>
+                              <Td>{t.therapeutic_agents || '--'}</Td>
+                              <Td>{t.treatment_intent_type || '--'}</Td>
+                              <Td>{t.treatment_or_therapy || '--'}</Td>
+                              <Td>{t.days_to_treatment || '--'}</Td>
+                            </Tr>
+                          )}
+                        </tbody>
+                      }
+                    />
                   }
-                />
-              }
-              {(!x.treatments || !x.treatments.length) &&
-                <div>No treatments found.</div>
-              }
-            </span>
-          )}
-        />
+                  {(!x.treatments || !x.treatments.length) &&
+                    <div>No Treatments found.</div>
```

Comment:
I think its because there is already a `<h3>` label above it.
![image](https://cloud.githubusercontent.com/assets/5777761/24017538/d31386c0-0a66-11e7-8183-f91366d44356.png)